src/runner.go

```diff
@@ -67,15 +66,31 @@ func verifyYamlFile(yamlData []byte) bool {
 	return true
 }
 
+func setEnvVariablesForCommand(cmd *exec.Cmd, variables map[string]string) *exec.Cmd {
+	// Set env variables
+	cmd.Env = os.Environ()
+	getEnvVarName := func(key string) string {
+		return fmt.Sprintf("RHC_WORKER_%s", strings.ToUpper(key))
+	}
+	for key, value := range variables {
+		prefixedKey := getEnvVarName(key)
+		envVarSetString := fmt.Sprintf("%s=%s", prefixedKey, value)
+		cmd.Env = append(cmd.Env, envVarSetString)
+		log.Infoln("Successfully set env variable ", envVarSetString)
+	}
+	return cmd
```

Comment:
Do we need to return the cmd? It seems to be that by passing the cmd as a parameter, we could modify the env property tied to it and we would not need to return it after we are done... maybe this is not how it works?

Reply:
Nope, removed